tools/hangar/__snapshots__/test_corpus/valid/api_cors_default.test.w_compile_tf-aws.md

```diff
@@ -289,126 +289,118 @@ module.exports = function({ $apiDefaultCors_url, $expect_Util, $http_HttpMethod,
-            $http_Util: ${$stdlib.core.liftObject($stdlib.core.toLiftableModuleType(http.Util, "@winglang/sdk/http", "Util"))},
-          })
-        `;
-      }
-      _toInflight() {
-        return `
-          (await (async () => {
-            const $Closure2Client = ${$Closure2._toInflightType(this)};
-            const client = new $Closure2Client({
-            });
-            if (client.$inflight_init) { await client.$inflight_init(); }
-            return client;
-          })())
-        `;
-      }
-      _supportedOps() {
-        return [...super._supportedOps(), "handle", "$inflight_init"];
-      }
-      _registerOnLift(host, ops) {
-        if (ops.includes("handle")) {
-          $Closure2._registerOnLiftObject(apiDefaultCors.url, host, []);
-        }
-        super._registerOnLift(host, ops);
-      }
+    static _toInflightType() {
+      return `
+        require("./inflight.$Closure1-1.js")({
+        })
+      `;
     }
-    class $Closure3 extends $stdlib.std.Resource {
-      _hash = require('crypto').createHash('md5').update(this._toInflight()).digest('hex');
-      constructor($scope, $id, ) {
-        super($scope, $id);
-        (std.Node.of(this)).hidden = true;
-      }
-      static _toInflightType() {
-        return `
-          require("./inflight.$Closure3-1.js")({
-            $apiDefaultCors_url: ${$stdlib.core.liftObject(apiDefaultCors.url)},
-            $expect_Util: ${$stdlib.core.liftObject($stdlib.core.toLiftableModuleType(expect.Util, "@winglang/sdk/expect", "Util"))},
-            $http_HttpMethod: ${$stdlib.core.liftObject($stdlib.core.toLiftableModuleType(http.HttpMethod, "@winglang/sdk/http", "HttpMethod"))},
-            $http_Util: ${$stdlib.core.liftObject($stdlib.core.toLiftableModuleType(http.Util, "@winglang/sdk/http", "Util"))},
-          })
-        `;
-      }
-      _toInflight() {
-        return `
-          (await (async () => {
-            const $Closure3Client = ${$Closure3._toInflightType(this)};
-            const client = new $Closure3Client({
-            });
-            if (client.$inflight_init) { await client.$inflight_init(); }
-            return client;
-          })())
-        `;
-      }
-      _supportedOps() {
-        return [...super._supportedOps(), "handle", "$inflight_init"];
+    _toInflight() {
+      return `
+        (await (async () => {
+          const $Closure1Client = ${$Closure1._toInflightType(this)};
+          const client = new $Closure1Client({
+          });
+          if (client.$inflight_init) { await client.$inflight_init(); }
+          return client;
+        })())
+      `;
+    }
+    _supportedOps() {
+      return [...super._supportedOps(), "handle", "$inflight_init"];
+    }
+  }
+  class $Closure2 extends $stdlib.std.Resource {
+    _hash = require('crypto').createHash('md5').update(this._toInflight()).digest('hex');
+    constructor($scope, $id, ) {
```

Comment:
would have been nice if these snapshots wouldn't have changed. it's a bit hard to tell why they did and it reduces my confidence in this pull request.

Reply:
I can undo it. I made the change to test reusing the `main` function for winglang codegen as well.

Reply:
undid those changes, it wasn't crucial for this PR